packages/core/src/types.ts

```diff
@@ -157,8 +177,8 @@ export interface Language {
 	 */
 	matchesSelectorClass?(
 		className: string,
-		node: SyntaxElement,
-		ancestry: Array<SyntaxElement>,
+		node: object,
+		ancestry: Array<object>,
```

Comment:
Using `object` instead of `Object` to enforce that this cannot be a primitive value.